library/src/services/form/Field/TextField/TextField.tsx

```diff
@@ -72,10 +72,16 @@ export const TextField = (props: TextFieldProps) => {
 
   passThroughProps.onChange = (event) => {
     const { target } = event;
-
+    console.log(event.target);
+    console.log({ ...event.target });
```

Comment:
Please remove this console.logs and set an eslint rule to avoid more console.log in project